### PR TITLE
Removed overly fancy terms events naming scheme

### PR DESCRIPTION
**Pull request by robertfairhead**
Follow-up on https://github.com/department-of-veterans-affairs/vets-website/pull/5941

Looking at it in practice @patrickvinograd was right and folks are coming in at sub-URLs so the fancy naming scheme isn't actually all that useful.

screen shot 2017-07-14 at 3 19 07 pm

Worth a shot, I guess.

Reverting to a simpler naming scheme before too much data piles up. We can use the pages view of events (as shown above) to determine what page caused the terms to be shown so there's real data loss from dropping it from the event name.

### CODE REVIEW

**Comment by ayaleloehr** [comment]
@robertfairhead is this just waiting on a review from someone on Kudos (likely @U-DON @saneshark or @patrickvinograd ) or is there more to be done?

**Comment by robertfairhead** [comment]
Yeah, just needs the review :+1:

This is a low-speed one as the change isn't particularly impactful. The event prefix will remain the same so it's easy to filter before/after. This is mostly a long-term cleanliness thing.

Was working with Patrick on some Facility Locator items that natalie wanted so this was back-burnered.

**Review by U-DON — APPROVED** [review]
LGTM.

**Pull request merged**
